### PR TITLE
automate publish to github package

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,65 +1,30 @@
 {
   "compilerOptions": {
-    /* Basic Options */
-    "incremental": true /* Enable incremental compilation */,
```
[the remark this answers is not in the document]
Comment (septs) [reply tsconfig.json:4]: need full output

File: package.json [hunk 2]
```diff
@@ -1,9 +1,14 @@
 {
   "name": "@dimensiondev/webextension-shim",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "repository": "https://github.com/DimensionDev/webextension-polyfill",
   "license": "MIT",
-  "main": "index.js",
+  "main": "./dist/index.js",
+  "module": "./es/index.js",
```
Comment (Jack-Works) [inline package.json:7]: This package is not designed to be import-able, it's not a library.

This library doesn't provide any public API (only polyfill for WebExtension API). It generates a single file "webextension-polyfill.js" and a modified version of "typescript.js". The native side should load it directly.

File: tsconfig.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jack-Works) [reply tsconfig.json:4]: If you don't have a `tsbuildinfo` file (on CI, it don't), it _will_ have a full output.

File: package.json (hunk 2 above)
Comment (septs) [reply package.json:7]: OK, remove `main` and `module`?

Comment (Jack-Works) [reply package.json:7]: Also the `tsc` building part. Only `rollup` is required.

File: src/bin/prebuilt.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```
Comment (Jack-Works) [inline src/bin/prebuilt.ts:1]: It's a ts file. Should use `ts-node`

File: src/bin/prebuilts.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```
Comment (Jack-Works) [inline src/bin/prebuilts.ts:1]: `ts-node`

File: src/bin/prebuilt.ts (hunk 3 above)
Comment (septs) [reply src/bin/prebuilt.ts:1]: image

Comment (Jack-Works) [reply src/bin/prebuilt.ts:1]: No need to compile it into JS, add complexity to the repo

Comment (septs) [reply src/bin/prebuilt.ts:1]: ???????

Comment (Jack-Works) [reply src/bin/prebuilt.ts:1]: I think it's okay to require users (the only user is Maskbook) to install the ts-node to run it.

Comment (septs) [reply src/bin/prebuilt.ts:1]: I think that this was unreasonable.

Comment (Jack-Works) [reply src/bin/prebuilt.ts:1]: ok, so remove the esm part, only leave the cjs part. Or bundle these two commands as a single file by rollup

Comment (septs) [reply src/bin/prebuilt.ts:1]: You come to implements it

File: package.json (hunk 2 above)
Comment (septs) [reply package.json]: Removed `main` and `module`.

File: src/bin/prebuilt.ts (hunk 3 above)
Comment (Jack-Works) [reply src/bin/prebuilt.ts:1]: Remove the esm part will be ok

**Pull request merged**